api/cloudprotocol/cloudprotocol.go

```diff
@@ -31,7 +31,7 @@ const ProtocolVersion = 6
 // ReceivedMessage structure for Aos incoming messages.
 type ReceivedMessage struct {
 	Header MessageHeader `json:"header"`
-	Data   []byte        `json:"data"`
+	Data   interface{}   `json:"data"`
```

Comment:
Probably we should use `json.Raw` here. Need to check how it will be used.

Reply:
changed. thank you.